include/mgp.hpp

```diff
@@ -2590,6 +2653,26 @@ inline bool Node::operator==(const Node &other) const { return util::NodesEqual(
 
 inline bool Node::operator!=(const Node &other) const { return !(*this == other); }
 
+inline const std::string Node::ToString() const {
+  std::string labels{""};
+  for (auto label : Labels()) {
+    labels += ":" + std::string(label) + " ";
+  }
+  std::string properties = "";
+  std::map<std::string, Value> properties_map{Properties()};
+  const size_t map_size = properties_map.size();
+  size_t i = 0;
+  for (auto item : properties_map) {
+    if (i == map_size - 1) {
+      properties += std::string(item.first) + "=" + item.second.ToString();
+      break;
+    }
+    properties += std::string(item.first) + "=" + item.second.ToString() + ", ";
+    ++i;
+  }
+  return "Node(id: " + std::to_string(Id().AsInt()) + " " + labels + "properties: {" + properties + "} )";
```

Comment:
I think you don't need to return `Node` here, so you can just do the following:

`(n:Label1:Label2 {prop1:"key", prop2:"key"}`